internal/featuresupport/feature_support_test.go

```diff
@@ -135,6 +136,59 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 		})
 	})
 
+	Context("Test skip MCO reboot", func() {
+		feature := models.FeatureSupportLevelIDSKIPMCOREBOOT
+		It("IsFeatureAvailable", func() {
+			Expect(IsFeatureAvailable(feature, "4.15", swag.String(models.ClusterCPUArchitecturePpc64le))).To(Equal(true))
+			Expect(IsFeatureAvailable(feature, "4.15", swag.String(models.ClusterCPUArchitectureX8664))).To(Equal(true))
+			Expect(IsFeatureAvailable(feature, "4.15", swag.String(models.ClusterCPUArchitectureS390x))).To(Equal(false))
+			Expect(IsFeatureAvailable(feature, "4.15", swag.String(models.ClusterCPUArchitectureArm64))).To(Equal(true))
+		})
+		It("GetSupportLevel on architecture", func() {
+			featureSupportParams := SupportLevelFilters{OpenshiftVersion: "4.15", CPUArchitecture: swag.String(models.ClusterCPUArchitectureX8664)}
+			Expect(GetSupportLevel(feature, featureSupportParams)).To(Equal(models.SupportLevelSupported))
+
+			featureSupportParams.CPUArchitecture = swag.String(models.ClusterCPUArchitectureS390x)
+			Expect(GetSupportLevel(feature, featureSupportParams)).To(Equal(models.SupportLevelUnavailable))
+
+			featureSupportParams.CPUArchitecture = swag.String(models.ClusterCPUArchitecturePpc64le)
+			Expect(GetSupportLevel(feature, featureSupportParams)).To(Equal(models.SupportLevelSupported))
+
+			featureSupportParams.CPUArchitecture = swag.String(models.ClusterCPUArchitectureArm64)
+			Expect(GetSupportLevel(feature, featureSupportParams)).To(Equal(models.SupportLevelSupported))
+		})
+		It("feature compatible with architecture", func() {
+			feature := &skipMcoReboot{}
+			openshiftVersion := "4.15"
+			for _, arch := range []string{
+				models.ClusterCPUArchitectureX8664,
+				models.ClusterCPUArchitectureArm64,
+				models.ClusterCPUArchitectureAarch64,
+				models.ClusterCPUArchitectureS390x,
+				models.ClusterCPUArchitecturePpc64le} {
+				Expect(isFeatureCompatibleWithArchitecture(feature, openshiftVersion, arch)).To(BeTrue())
```

Comment:
you removed the call for this function is that test still valid?

Reply:
It may be called (outside my code)